GitUI/CommitInfo/RefsFormatter.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using GitCommands;
+using ResourceManager;
+
+namespace GitUI.CommitInfo
+{
+    public sealed class RefsFormatter
```

Comment:
If a class contains only `static` members, the class should be marked as `static` (a small perf optimisation).
However can we not make members static? It creates coupling and makes testing harder

Reply:
Yes, it had been a `static` class until I implemented `ITranslate`. Forgot to turn it back to `static`.

This class contains only static methods and an instance of `LinkFactory`. But it does not use stateful methods of `LinkFactory`.